test/e2e/setup-infra.sh

```diff
@@ -188,7 +188,7 @@ function install_turing() {
     cd "$script_dir/../.."
 
     kubectl apply -f ./test/e2e/turing.mockserver.yaml
-    helm install turing ./infra/chart -f ./test/e2e/turing.helm-values.yaml --wait
+    helm upgrade --install turing ./infra/chart -f ./test/e2e/turing.helm-values.yaml --wait
```

Comment:
What is the rational?

Reply:
I'll remove this, I remember now. There was one run where the helm release couldn't install because it had the same release name. In retrospect I think it was because i called it twice. Will revert this